packages/driver/cypress/integration/issues/8998_spec.js

```diff
@@ -0,0 +1,8 @@
+it('t', () => {
```

Comment:
Fixing test title to more easily track this down if it fails.

```suggestion
// https://github.com/cypress-io/cypress/issues/8998
it('issue 8998', () => {
```